crates/sui/src/client_commands.rs

```diff
@@ -665,22 +653,20 @@ impl SuiClientCommands {
         context: &mut WalletContext,
     ) -> Result<SuiClientCommandResult, anyhow::Error> {
         let ret = Ok(match self {
-            SuiClientCommands::ReplayTransaction { rpc_url, tx_digest } => {
+            SuiClientCommands::ReplayTransaction { tx_digest } => {
                 let cmd = ReplayToolCommand::ReplayTransaction {
                     tx_digest,
                     show_effects: true,
                     diag: false,
                     executor_version_override: None,
                     protocol_version_override: None,
                 };
-                // todo: automatically get non pruning fullnode url
+                let rpc = context.config.get_active_env()?.rpc.clone();
                 let _command_result =
-                    sui_replay::execute_replay_command(Some(rpc_url), false, false, None, cmd)
-                        .await;
+                    sui_replay::execute_replay_command(Some(rpc), false, false, None, cmd).await;
```

Comment:
This command produces a result -- should we not check whether it errored? Currently we're ignoring its results.